scripts/create_index.js

```diff
@@ -6,6 +6,15 @@ const SUPPORTED_ES_VERSIONS = '>=7.4.2';
 const cli = require('./cli');
 const schema = require('../schema');
 
+// Check that the shape_type property has been defined in the schema.js file
+// This must be set to either 'shape' or 'polygon', if not exit
+if(shapeType === 'shape' || shapeType === 'polygon'){
+  console.log('\n Creating an ES index with schema shape type: ' + shapeType + '.');
```

Comment:
Hi Aine, you might be able to use template literal here with backticks, that way you can avoid string concat and might look a bit cleaner, up to you though. 
`
  console.log(`\n Creating an ES index with schema shape type: ${shapeType}.`);
`

Reply:
Cheers Alen I updated that there now.